requests/compat.py

```diff
@@ -64,3 +65,4 @@
     bytes = bytes
     basestring = (str, bytes)
     numeric_types = (int, float)
+    integer_types = (int)
```

Comment:
This is a common tricky area, but `(int)` doesn't actually make a tuple. In this case, you'd want `(int,)`.